splink/datasets/__init__.py

```diff
@@ -31,22 +31,120 @@ def __post_init__(self):
             )
 
 
-_splink_demo_data_dir = (
-    "https://raw.githubusercontent.com/moj-analytical-services/splink_demos/master/data"
-)
+_splink_datasets_data_dir = """https://raw.githubusercontent.com/moj-analytical-services
+                                /splink_datasets/master/data"""
 _datasets = [
     _DataSetMetaData(
         "fake_1000",
-        f"{_splink_demo_data_dir}/fake_1000.csv",
+        f"{_splink_datasets_data_dir}/fake_1000.csv",
         # hard code metadata to avoid needing to download to build docs
-        1000,
+        "1,000",
         250,
         (
             "Fake 1000 from splink demos.  "
             "Records are 250 simulated people, "
             "with different numbers of duplicates, labelled."
         ),
     ),
+    _DataSetMetaData(
+        "fake_1000_labels",
+        f"{_splink_datasets_data_dir}/fake_1000_labels.csv",
+        # hard code metadata to avoid needing to download to build docs
+        "1,225",
+        "NA",
+        ("Clerical labels for pairwise comparison  " "from fake_1000 "),
+    ),
+    _DataSetMetaData(
+        "labels_to_estimate_m",
+        f"{_splink_datasets_data_dir}/pairwise_labels_to_estimate_m.csv",
+        # hard code metadata to avoid needing to download to build docs
+        "2,032",
+        "NA",
+        (
+            "Clerical labels for pairwise comparison  "
+            " for model training from fake_1000 "
+        ),
+    ),
```

Comment:
This might be overcomplicating things, but I wonder if these should live on a separate object (`splink_datasets_labels` or sth) to keep the different usages separate? Could even do the same for dedupe sets vs link sets, but that might be overkill

Reply:
Also, looking at these datasets themselves they seem functionally pretty similar - do we need both, or could we use one for both functions?

Reply:
Yea, a separate section for labels sounds sensible. I had thought that these looked v similar, but didn't dig into how we could use a single dataset for both cases. Will have a look and report back

Reply:
Hey @ADBond I have pushed a version that has separated out the labels. Do you think this is sufficient? Or should we have a completely separate structure for the labels datasets? I have just reused all of the structures you have created with everything still being a `_DataSetMetaData` class.

I am working on the choice between labels dataset separately